chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/avatar-group/README.md

# @vaadin/avatar-group

> A web component used to group multiple [`<vaadin-avatar>`](https://www.npmjs.com/package/@vaadin/avatar-group) components together.

Reviewer: ```suggestion
A web component for grouping multiple [`<vaadin-avatar>`](https://www.npmjs.com/package/@vaadin/avatar-group) components together.
```

Author: Done.
